tests/ai/openai/test_openai_prompter.py

```diff
@@ -739,6 +739,49 @@ async def _test():
     run_async(_test())
 
 
+def test_openai_prompter_with_text_file():
+    """Test prompting with a plain text document."""
+
+    async def _test():
+        import os
+        import tempfile
+
+        mock_client = AsyncMock()
+        mock_response = Mock()
+        mock_response.output_text = "This appears to be a text document."
+        mock_client.responses.create = AsyncMock(return_value=mock_response)
+
+        prompter = OpenAIPrompter(
+            provider_options={"api_key": "test-key"},
+            model="gpt-4o-mini",
+        )
+        prompter.llm = mock_client
+
+        with tempfile.NamedTemporaryFile(suffix=".txt", delete=False, mode="w", encoding="utf-8") as tmp:
+            tmp.write("This is a plain text document.")
+            temp_path = tmp.name
+
+        try:
+            from daft.file import File
+
+            result = await prompter.prompt(("Summarize this document.", File(temp_path)))
+
+            assert result == "This appears to be a text document."
+
+            call_args = mock_client.responses.create.call_args
+            messages = call_args.kwargs["input"]
+            assert len(messages) == 1
+            assert isinstance(messages[0]["content"], list)
+            assert len(messages[0]["content"]) == 2
+            text_message = messages[0]["content"][1]
+            assert text_message["type"] == "input_text"
+            assert text_message["text"] == "File contents: This is a plain text document."
```

Comment:
Extending my suggestion to the test: 
```suggestion
            assert text_message["text"] == "<file_text_plain>This is a plain text document.</file_text_plain>"
```